types/package.json

```diff
@@ -30,6 +30,11 @@
     "dts-cli": "^2.0.3",
     "eventsource-parser": "^1.1.1",
     "io-ts": "^2.2.20",
+    "redis": "^4.6.8",
     "uuid": "^9.0.1"
+  },
+  "browser": {
```

Comment:
please give us a crash course on this in the PR comments :)

Reply:
My understanding is that there is a build step in nextjs that was trying to package the file importing Redis in a front-end code, which is not possible, and adding that tells the builder (webpack apparently) to not package these for client side code.

Unfortunately, online documentation is really poor on this, so I can't provide a super clear answer.  But everything works well.

This is the most plausible answer I found [online](https://stackoverflow.com/a/74240581):

When I added redis as a dependency to `types/`, front build started to fail with the following errors:
```
Module not found: tls
Module not found: net
```
The solution to this error was to specify that.

Reply:
OK let's give it a try. I worry we might be missing with SSR here... but we'll know soon enough. Let's just remind ourselves to revisit this if we feel there are performance regressions in the coming days